repose-aggregator/external/pjl-compressing-filter/src/test/java/org/openrepose/external/pjlcompression/CompressingFilterResponseTest.java

```diff
@@ -368,13 +370,13 @@ public void doGet(HttpServletRequest request,
             byte[] outputBytes = output.getBytes(TEST_ENCODING);
             byte[] expectedBytes = getCompressedOutput(outputBytes);
             byte[] moduleOutput = response.getContentAsByteArray();
-            assertFalse(Arrays.equals(outputBytes, moduleOutput));
+            assertThat(outputBytes, not(equalTo(moduleOutput)));
             assertArrayEquals(expectedBytes, moduleOutput);
             assertEquals(Boolean.TRUE, request.getAttribute(CompressingFilter.COMPRESSED_KEY));
 
             assertTrue(response.containsHeader("Content-Encoding"));
             assertTrue(response.containsHeader("X-Compressed-By"));
-            assertTrue(!response.containsHeader("ETag") || response.getHeader("ETag").endsWith("-gzip"));
+            assertThat(response.getHeader("ETag"), anyOf(org.hamcrest.Matchers.nullValue(), org.hamcrest.Matchers.endsWith("-gzip")));
```

Comment:
Those fully qualified methods really make me want to drop the Mockito Matchers that I assuming are forcing this. That ties back to an earlier comment I made about always preferring Hamcrest Matchers.